Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG BASE_IMAGE=rapidsai/rapidsai:cuda11.0-runtime-ubuntu18.04-py3.8
+
+FROM ${BASE_IMAGE}
+RUN apt update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    git python3-setuptools python3-pip build-essential
+
+RUN /opt/conda/envs/rapids/bin/pip install \
+    scanpy wget python-igraph louvain leidenalg scanpy
```

Comment:
1. Are you sure that all dependencies in `https://github.com/clara-parabricks/rapids-single-cell-examples/blob/master/conda/rapidgenomics_cuda10.2.yml` are included in the container? I see only a few of them being installed here.

2. `leidenalg` is not included in `https://github.com/clara-parabricks/rapids-single-cell-examples/blob/master/conda/rapidgenomics_cuda10.2.yml` but it is included here. Is it necessary? Why?

Reply:
All these notebooks were retested on AWS yesterday to make sure this change does not break any of the notebook.